libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -321,11 +385,12 @@ export class OperationExecutionManager {
     // Apply status changes to direct dependents
     for (const item of record.consumers) {
       if (blockCacheWrite) {
-        item.runner.isCacheWriteAllowed = false;
+        item.isCacheWriteAllowed = false;
```

Comment:
In this context, "blockCacheWrite" might be true if the operation failed, right? In previous implementation this also meant no downstream operations could be written to cache. Does this change that behavior?